app.js

```diff
@@ -50,7 +70,13 @@ console.log(
 //    that checks if the year is a leap year, and returns the correct
 //    number of days for Feb. The default value for the year should be
 //    the current year
-function daysInMonth() {}
+let daysOfMonth = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+function daysInMonth(month = '2022') {
```

Comment:
This works but it is confusing to read with '2022' being in there. I suspect you were about to go for the bonus question and check for leap years but didn't quite make it. JavaScript then overwrites the '2022' with the number input in the console.log's below. Keep an eye out for these things since they can cause issues down the road.